### PR TITLE
Edit support ticket

### PR DESCRIPTION
**Pull request by ketankartoza**

### CODE REVIEW

File: geohosting/api.py [hunk 1]
```diff
@@ -48,6 +48,23 @@ def get_csrf_token():
     return {
         "csrf_token": frappe.sessions.get_csrf_token()
     }
-    
 
+@frappe.whitelist(allow_guest=False)
```
Comment (tinashechiraya) [inline geohosting/api.py:52]: @ketankartoza can you move this method to support tickets.py and call it from there

File: geohosting/www/portal/support_tickets.html [hunk 2]
```diff
@@ -12,6 +12,56 @@
 <script src="/assets/geohosting/js/util.js"></script> 
 <style>
     
+.card {
```
Comment (tinashechiraya) [inline geohosting/www/portal/support_tickets.html:15]: @ketankartoza can you also not use a modal to edit the ticket rather use the existing form for creating a ticket . So when user click on ticket it should repeat the action similar to when user clicks new ticket ...then on that form opened show the details of the ticket being edited

File: geohosting/api.py (hunk 1 above)
Comment (ketankartoza) [reply geohosting/api.py:65]: ok

File: geohosting/www/portal/support_tickets.html [hunk 3]
```diff
@@ -752,7 +773,74 @@ <h4>${message}</h4>
...
+          "raised_by": name,
+          "owner": "{{ frappe.session.user }}",
+          "subject": subject,
+          "priority": priority,
+          "issue_type": issueType,
+          "description": editorData,
+          "status": "Open"
+      };
+
+      console.log(requestBody);
```
Comment (tinashechiraya) [inline geohosting/www/portal/support_tickets.html:800]: @ketankartoza can you remove console logs

File: geohosting/www/portal/support_tickets.html [hunk 4]
```diff
@@ -752,7 +773,74 @@ <h4>${message}</h4>
...
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(requestBody)
+      }
+
+        const url = `/api/method/geohosting.www.portal.support_tickets.update_issue`;
+
+        await fetchWithCSRF(url, options)
+        .then(response => {
+            console.log('Fetch response received:', response); // Add logging
```
Comment (tinashechiraya) [inline geohosting/www/portal/support_tickets.html:814]: @ketankartoza can you remove

**Review by tinashechiraya — APPROVED** [review]

**Pull request merged**